### PR TITLE
feat: add banzai logging operator support

### PR DESCRIPTION
**Pull request by brewkode**
Hi folks,

I added support for banzai logging operator crds. All but the "logging.banzaicloud.io_loggings.yaml" had some issues, which I have skipped for now. Kindly review and merge to upstream.

### CODE REVIEW

File: libs/banzai-logging/config.jsonnet [hunk 1]
```diff
@@ -0,0 +1,21 @@
+local config = import 'jsonnet/config.jsonnet';
+
+local versions = [
+  { output: '3.17.3', version: '3.17.3' },
+];
+
+local crds = ['logging.banzaicloud.io_flows.yaml', 'logging.banzaicloud.io_clusterflows.yaml', 'logging.banzaicloud.io_clusteroutputs.yaml', 'logging.banzaicloud.io_outputs.yaml'];
```
Comment (Duologic) [inline libs/banzai-logging/config.jsonnet:7]: Can you add some linebreaks here? Makes future changes to this list easier to review.

Comment (brewkode) [reply libs/banzai-logging/config.jsonnet]: Sure, done!

**Comment by brewkode** [comment]
@Duologic is there a general release cadence for this project or is it per PR Merge ?

**Comment by Duologic** [comment]
Just per PR merge, it automatically propagates to the different GitHub repos on merge.

**Pull request merged**